server/app/controllers/ti/TrustedIntermediaryController.java

```diff
@@ -110,36 +113,30 @@ public Result updateDateOfBirth(Long tiId, Long applicantId, Http.Request reques
     if (trustedIntermediaryGroup.isEmpty()) {
       return notFound();
     }
-    if (!trustedIntermediaryGroup.get().id.equals(tiId)) {
-      return unauthorized();
-    }
-    Form<UpdateApplicantDOB> form =
-        formFactory.form(UpdateApplicantDOB.class).bindFromRequest(request);
-    if (form.hasErrors()) {
-      return redirectToDashboardWithUpdateDateOfBirthError(form.errors().get(0).message(), form);
-    }
-    if (Strings.isNullOrEmpty(form.get().getDob())) {
-      return redirectToDashboardWithUpdateDateOfBirthError("Date Of Birth is required.", form);
-    }
-
+    Form<UpdateApplicantDob> form =
+        formFactory.form(UpdateApplicantDob.class).bindFromRequest(request);
     try {
-      Applicant applicant =
-          userRepository.lookupApplicant(applicantId).toCompletableFuture().join().get();
-      applicant.getApplicantData().setDateOfBirth(form.get().getDob());
-      userRepository.updateApplicant(applicant);
-    } catch (DateOfBirthExistsException e) {
-      String trustedIntermediaryUrl = baseUrl + "/trustedIntermediaries";
-
-      return redirectToDashboardWithUpdateDateOfBirthError(
-          "Date Of birth for this applicant exists.  Cannot Update DOB. "
-              + " Direct applicant to sign in and go to"
-              + " "
-              + trustedIntermediaryUrl,
-          form);
+      tiService.checkFormForDobUpdate(form, applicantId);
```

Comment:
It appears that this method also updates the applicant data rather than just "checking it". Perhaps this could be renamed to `updateApplicantDateOfBirth`?

Also, minor nit on service argument order. Typically the database ID for the entity to be updated is the first argument.